pkg/kontainer-engine/drivers/gke/gke_driver.go

```diff
@@ -883,27 +877,19 @@ func getTokenSource(ctx context.Context, state state) (oauth2.TokenSource, error
 	if err != nil {
 		return nil, err
 	}
-	return ts, nil
-}
 
-func getServiceClientWithTokenSource(ctx context.Context, tokenSource oauth2.TokenSource) (*raw.Service, error) {
-	client := oauth2.NewClient(ctx, tokenSource)
-	service, err := raw.NewService(ctx, option.WithHTTPClient(client))
-	if err != nil {
-		return nil, err
-	}
-	return service, nil
-}
+	// Unlocks
+	cleanup()
 
-func getServiceClient(ctx context.Context, state state) (*raw.Service, error) {
-	ts, err := getTokenSource(ctx, state)
+	client := oauth2.NewClient(ctx, ts)
+	service, err := raw.New(client)
```

Comment:
This part looks oddly different from the original PR ? https://github.com/rancher/rancher/pull/29818/files#diff-2ba6ad7c3d997ca9d256feab9fe0e3439a5804815e98eec920daf27531113e5eL885

Reply:
I'm not sure what you mean. It looks the same to me.